xml/ha_rear.xml

```diff
@@ -24,7 +24,7 @@
       <abstract>
         <para>
     &rear.long; (formerly <quote>ReaR</quote>, in this chapter
-    abbreviated as &rear;) is a disaster recovery framework for use by
```

Comment:
I still see the word 'formerly' in this pull request
in its "Files changed" tab.
Perhaps you need to 'git push' your local changes?